tests/test_s3_specific.py

```diff
@@ -164,6 +164,36 @@ def test_fake_directories(s3_like_rig):
         assert not test_case.is_file()
 
 
+def test_directories(s3_like_rig):
+    """Fixing bug that marks as directories prefixes of existing paths
+
+    Ref: https://github.com/drivendataorg/cloudpathlib/issues/208
+    """
+    boto3_s3_client = s3_like_rig.client_class._default_client.client
+    if not s3_like_rig.live_server:
+        pytest.skip("This test only runs against live servers.")
+
+    response = boto3_s3_client.put_object(
+        Bucket=f"{s3_like_rig.drive}",
+        Body="123",
+        Key=f"{s3_like_rig.test_dir}/superfile",
+    )
+
+    assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
```

Comment:
I think that we can just use one of the files that gets populated by the rig automatically (for example, like we do here):
https://github.com/drivendataorg/cloudpathlib/blob/2e52c0515547e998e0b7c84e667627df4ac332b6/tests/test_cloudpath_file_io.py#L17-L19